packages/firebase_analytics/CHANGELOG.md

```diff
@@ -1,10 +1,33 @@
-## 1.2.0
+## 2.0.0
+
 * **Breaking change**. Migrate from the deprecated original Android Support
   Library to AndroidX. This shouldn't result in any functional changes, but it
   requires any Android apps using this plugin to [also
   migrate](https://developer.android.com/jetpack/androidx/migrate) if they're
   using the original support library.
 
+  This was originally incorrectly pushed in the `1.2.0` update.
```

Comment:
Maybe we could add some notes or a link here about how to properly upgrade for it?

Reply:
I'm already linking to the Android docs on how to upgrade using Android Studio. I could also write a page in our wiki with extra steps on how to import the project into Android Studio, or a guide on what to do if you want to migrate manually? Or do you think the the link on how to migrate above is too hidden? 

I'm a little wary about blocking these PRs on having a good guide written up, but I could probably at least start a page and get a URL and just forward to the official Android docs for now, then fill it out better in parallel.

Reply:
That link looks good enough to me.